applications/nrf5340_audio/src/bluetooth/bt_stream/bt_le_audio_tx/bt_le_audio_tx.c

```diff
@@ -103,23 +111,51 @@ static int iso_stream_send(uint8_t const *const data, size_t size, struct bt_bap
 	net_buf_reserve(buf, BT_ISO_CHAN_SEND_RESERVE);
 	net_buf_add_mem(buf, data, size);
 
-	atomic_inc(&tx_info->iso_tx_pool_alloc);
+	atomic_inc(&tx_inf_out->iso_tx_pool_alloc);
+	uint16_t seq_num;
+
+	if (tx_sync_get_info->sync_get_info_updated) {
+		if ((tx_inf_out->iso_tx.seq_num != tx_sync_get_info->iso_tx.seq_num) &&
+		    (tx_inf_out->iso_tx.seq_num != 1)) {
+			{
+				/* An ISO TX window has been missed. I.e. the application layer sent
+				 * the data too late to make it into the first/ideal ISO interval.
+				 * Depending on the configuration of the ISO link, this may result
+				 * in either flushed data, or it may be re-transmitted in the next
+				 * ISO interval. Errors here may indicate:
+				 * - Incoming Audio data (e.g. I2S or USB) not being kept in sync
+				 * with the BLE clock. The latter requires asynchronous USB support.
+				 * - Audio encoding taking too much time.
+				 * - Too high CPU load.
+				 */
```

Comment:
Most probably, the data will be scheduled for transmission in the next "ISO TX window". It would result in higher latency.

Reply:
True. I removed this check due to complexity. I created a task for it in our backlog (OCT-2913).